code/game/objects/items/kirbyplants.dm

```diff
@@ -63,11 +63,24 @@
 
 
 /obj/item/kirbyplants/dead
-	name = "RD's potted plant"
-	desc = "A gift from the botanical staff, presented after the RD's reassignment. There's a tag on it that says \"Y'all come back now, y'hear?\"\nIt doesn't look very healthy..."
+	name = "dead potted plant"
+	desc = "The unidentifiable plant remnants make you feel like planting something new in the pot."
 	icon_state = "plant-25"
 	trimmable = FALSE
 
+/obj/item/kirbyplants/dead/rd
+	name = "RD's potted plant"
+	desc = "A gift from the botanical staff, presented after the RD's reassignment. There's a tag on it that says \"Y'all come back now, y'hear?\"\nIt doesn't look very healthy..."
+
+/obj/item/kirbyplants/dead/attackby(obj/item/I, mob/living/user, params)
+	. = ..()
+	if(istype(I, /obj/item/seeds))
+		to_chat(user,span_notice("You start planting a new seed into the pot."))
+		if(do_after(user,3 SECONDS,target=src))
+			new /obj/item/kirbyplants(get_turf(src))
```

Comment:
This method of handling things won't copy certain things over - spraycan colors, if it's haunted, hand labeling... not really that important though.

Reply:
what do you suggest to use instead?

Reply:
Maybe a plant being dead can be a var on regular potted plants, updating icon when it dies/gets revived. Dead plants can just spawn with it dead.
That way we can also easily add plants dying too

Reply:
I wanted to refactor hydrotrays to have a single reagent tanks and turn water into a nutrient that just sustains plants but doesn't modify stats. Every potted plant would be a proper plant with seeds that can be grown in hydroponics. Pots would be a subtype of hydrotrays with a lot of water roundstart. And the potted plants would die if Janitor doesn't water them. Janitor would use service HUD to see the water level in the pot and the health of plants via #75279 And you could replace the potted plant with say tomatoes and people would be able to gather them in halls.

But in scope of this PR I just wanted to provide a simple way of turning dead plants into beautiful sprite because it was allowed with bonsai skill before, but was taken away.

Reply:
absolutely love the idea, though since we don't have that PR yet let's just make it a var that affects what happens in update_icon(). 

once you open the hydrotray pr (again i absolutely FUCKING LOVE the idea), it'd likely just be a var that affects update_icon as well too, wouldn't it? lol

Reply:
> once you open the hydrotray pr (again i absolutely FUCKING LOVE the idea), it'd likely just be a var that affects update_icon as well too, wouldn't it? lol

I'm not sure I'll ever do this. Melbert didn't like some parts of the design I don't have alternative to and he's wiser than me in botany.